src/flags.rs

```diff
@@ -32,6 +33,7 @@ pub fn print_usage() {
 --allow-env        Allow environment access.
 -v or --version    Print the version.
 -r or --reload     Reload cached remote resources.
+-c or --compile    Force recompilation of TypeScript code.
```

Comment:
Taking the -c flag seems unnecessary, is this something users are going to be doing often enough to merit this abbreviation?

Reply:
It’s a good point. We should try to be conservative with the number of flags added. Maybe —recompile is better?

Reply:
I agree, I've used `-c` and `--compile` as per issue.